mautic/mautic
# Add support for a channel subscription webhook

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Enhancement
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

This PR adds support to notify a webhook when a contact's channel subscription changes. 

[//]: # ( As applicable: )

#### Steps to test this PR:
1. Create a new webhook and check to send on channel subscription change using https://requestb.in/ or equivalent service
2. Go to the Contacts page and check a contact, click the bulk action drop down and click do not contact
3. Submit the form that opens in a modal
4. The webhook should be delivered
